test/e2e/shoot/internal/access_csr.go

```diff
@@ -20,6 +20,12 @@ import (
 	"crypto/x509/pkix"
 	"time"
 
+	"github.com/gardener/gardener/pkg/client/kubernetes"
```

Comment:
> On the way, the logic of `CleanupObjectsFromCSRAccess` was made reusable since it will be needed also for the created `ServiceAccount`s and related resources.

Did I miss something? I couldn't find this in the diff.

Reply:
I dropped it later on again 😄